src/commands/workers/deploy.ts

```diff
@@ -113,6 +116,33 @@ export default class Deploy extends BaseCommand<typeof Deploy> {
       workersConfig,
     });
 
+    const errorMessages = uploadDeployArg.workers.reduce<Array<string>>(
+      (acc, { config: { services }, hosts, name }) => {
+        if (services.length === 0) {
```

Comment:
I think this once can be rewritten as a chain of `map` and `filter` as well. Just return `null` if you don't want an item to be included into resulting collection and then filter out `null` elements